resources/testharness.js

```diff
@@ -568,7 +568,21 @@ policies and contribution forms [3].
         var test_name = name ? name : test_environment.next_default_test_name();
         var test_obj = new Test(test_name, properties);
         if (func) {
-            test_obj.step(func, test_obj, test_obj);
+            let value = test_obj.step(func, test_obj, test_obj);
```

Comment:
```suggestion
            var value = test_obj.step(func, test_obj, test_obj);
```

For consistency with surrounding code.